hooks/post-command

```diff
@@ -25,6 +25,135 @@ plugin_read_list_into_result() {
   [[ ${#result[@]} -gt 0 ]] || return 1
 }
 
+
+get_os() {
```

Comment:
Given that the uploader is only available for plain Linux, alpine and Macos (and windows which we don't currently support) can we simplify this one? If it just returned  the strings `linux`, `alpine` or `macos` we can feed that directly into the download URL

Reply:
Fixed in https://github.com/joscha/codecov-buildkite-plugin/commit/388c47dc57d487f625544c2c9532c6b66cc2b2e2